extensions/PhabBugz/lib/Feed.pm

```diff
@@ -111,15 +111,15 @@ sub feed_query {
         my $object_phid = $story_data->{objectPHID};
         my $story_text  = $story_data->{text};
 
-        DEBUG("STORY ID: $story_id");
-        DEBUG("STORY PHID: $story_phid");
-        DEBUG("AUTHOR PHID: $author_phid");
-        DEBUG("OBJECT PHID: $object_phid");
-        INFO("STORY TEXT: $story_text");
+        TRACE("FEED: STORY ID - $story_id");
```

Comment:
One thing, you could put info into Bugzilla::Logging->fields->{ITEM},
if you wanted to associate information with each log entry.

This means when you log an INFO()  or something in production, the information would be available in the 'Fields' JSON field.

Reply:
I like that idea so the new patch does that. Much cleaner IMO.